docs/guides/infineon_trustm_provisioning.md

```diff
@@ -28,21 +33,28 @@ The [Linux Tools for  OPTIGA&trade; Trust M ](https://github.com/Infineon/linux-
  $ ./trustm_installation_aarch64_script.sh
 ```
 
--   Run the script to generate Matter test DAC for lock-app using the public key extracted from the Infineon pre-provisioned Certificate and store it into 0xe0e3
+-   Run the script to generate Matter test DAC for lock-app using the public key
+    extracted from the Infineon pre-provisioned Certificate and store it into
+    0xe0e3
 
 ```
 $ cd scripts/matter_provisioning/
 $ ./matter_dac_provisioning.sh
 ```
 
-*Note:*
+_Note:_
 
-*By running this example matter_dac_provisioning.sh, the steps shown below are executed:*
+_By running this example matter_dac_provisioning.sh, the steps shown below are
+executed:_
 
- *Step1: Extract the public key from the Infineon pre-provisioned Certificate(0xe0e0) using openssl command.*
+_Step1: Extract the public key from the Infineon pre-provisioned
+Certificate(0xe0e0) using openssl command._
 
- *Step2: Generate DAC test certificate using the extracted public key, Signed by [Matter test PAI](https://github.com/project-chip/connectedhomeip/blob/v1.1-branch/credentials/development/attestation/Matter-Development-PAI-FFF1-noPID-Cert.pem)*
+_Step2: Generate DAC test certificate using the extracted public key, Signed by
```

Comment:
Add a note that production devices cannot re-use these test keys/certificates.

Reply:
Hi @tcarmelveilleux, Thanks for your review. we added a note for this in the latest commit.